cclunit-framework-tests/src/test/resources/META-INF/spring/applicationContext-cclutTests.xml

```diff
@@ -8,7 +8,7 @@ http://www.springframework.org/schema/context http://www.springframework.org/sch
 
     <!-- ftp-util configuration -->
     <bean factory-bean="ftpProductBuilder" factory-method="build" />
-    <bean id="ftpProductBuilder" class="com.cerner.ccl.testing.framework.internal.UserPassBuilderSpringProxy" p:password="#{systemProperties['ccl-hostPassword']}" p:username="#{systemProperties['ccl-hostUsername']}" p:serverAddress="${ccl-host}">
```

Comment:
Please revert the changes to the cclut-framework-tests project. 

Although they allow the build to work on devices which only have ccl-hostUsername and ccl-hostPassword values configured, they prevent the build from working on devices which only have ccl-hostCredentialsId set up which is how the continuous integration box used to deploy the framework code is set up.  

A change that works for both configurations would be okay.

Reply:
Oops, yeah, this was a mistake; I didn't mean to include this in the commit.  It was just part of the initial troubleshooting I was doing.